swagger.yaml

```diff
@@ -2533,6 +2533,10 @@ definitions:
         type: string
         description: Json formatted string containing the user overrides for the initial ignition config
         example: '{"ignition": {"version": "3.1.0"}, "storage": {"files": [{"path": "/tmp/example", "contents": {"source": "data:text/plain;base64,aGVscGltdHJhcHBlZGluYXN3YWdnZXJzcGVj"}}]}}'
+      connectivity_majority_groups:
+        type: string
+        description: Json formatted string containing the majority groups for conectivity checks.
+        x-go-custom-tag: gorm:"type:varchar(2048)"
```

Comment:
Do we know how many hosts/networks this can support?  What happens if we have overflow?

Reply:
Will change to text